homeassistant/components/huawei_lte/button.py

```diff
@@ -50,10 +51,7 @@ async def async_update(self) -> None:
     def press(self) -> None:
         """Press button."""
         if self.router.suspended:
-            _LOGGER.debug(
-                "%s: ignored, integration suspended", self.entity_description.key
-            )
-            return
+            raise ServiceValidationError("Integration is suspended")
```

Comment:
TBH I'm not sure if this (ditto in select and switch) is required/that useful now that the interactive entities' availability is updated based on if the parent router is suspended, but I guess it doesn't hurt.